pom.xml

```diff
@@ -17,6 +17,7 @@
         <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
         <structured-logging.version>1.9.14</structured-logging.version>
         <log4j.version>2.17.2</log4j.version>
+        <org.mapstruct.version>1.5.0.RC1</org.mapstruct.version>
```

Comment:
We used version 1.4.2 on Confirmation Statement but used the latest version available here.